packages/overlay/src/overlay-stack.ts

```diff
@@ -142,7 +230,7 @@ export class OverlayStack {
 
     private async hideAndCloseOverlay(
         overlay?: ActiveOverlay,
-        animated = true
+        animated?: boolean
```

Comment:
Is there a distinction between `animated: false` and `animated: null` for overlays?

Reply:
Technically, yes. The `overlay.hide(animated);` that receives this argument also does `animated = true` in its method signature, which means that `undefined` in this method actually means `true` as it did before when given a default, only this way we can get test coverage on both lines...